app/controllers/static_pages_controller.rb

```diff
@@ -0,0 +1,6 @@
+class StaticPagesController < ApplicationController
```

Comment:
oic, okay i think that's ok for now. 

**In the future** though I think we could kick this can down the road when we start implementing pages. Reason I say that is now whoever starts building the pages out will have to reassign the `root` and then remove static controller, views etc

`YAGNI` You ain't going to need it =)